pkgs/games/openra/engines/bleed/default.nix

```diff
@@ -0,0 +1,14 @@
+{ fetchFromGitHub }:
+
+rec {
+  name = "bleed";
+  version = "20230318";
```

Comment:
```suggestion
  pname = "bleed";
  version = "unstable-2023-03-18";
```

Reply:
see below

Reply:
Notice the meta section is filled out, in the `build-engine.nix`